solutions/object_oriented_design/parking_lot/parking_lot.py

```diff
@@ -85,9 +85,7 @@ def spot_freed(self):
 
     def park_vehicle(self, vehicle):
         spot = self._find_available_spot(vehicle)
-        if spot is None:
-            return None
-        else:
+        if spot:
```

Comment:
I do agree this is an optimized option, but the other version is a bit more explicit which may help with the readability of it.